packages/radix-ui-themes/src/components/tooltip.css

```diff
@@ -2,6 +2,7 @@
   padding: var(--space-1) var(--space-2);
   background-color: var(--gray-12);
   border-radius: var(--radius-2);
+  position: relative;
```

Comment:
I was on the fence about the usecase for this one, assuming most folks are using text in tooltips. I could see potentially using an absolutely positioned icon though. Open to thoughts.

Reply:
Yeah I can see what you mean, doesn't hurt to have it though.